Makefile.PL

```diff
@@ -525,12 +525,23 @@ USAGE
 }
 
 
+sub Execute {
+  my ($command) = @_;
+  # Silence "Can't exec" warnings
+  local $SIG{__WARN__} = sub {};
+  my $output = `$command 2>&1`;
+  # Detect command not found on Windows
+  $output = undef if $^O eq 'MSWin32' and defined $output and $output =~ /is not recognized as/;
```

Comment:
Are all of `$?`, `$^E`, and `${^CHILD_ERROR_NATIVE}` ignored?

Reply:
`$?` is exit code, so not suitable
`$^E` is value of the last `GetLastError()` WINAPI call, nothing interesting
`${^CHILD_ERROR_NATIVE}` seems like extended exit code

As `$output` contains something, it means that process started and successfully finished. Documentation for backtick operator says: `A string which is (possibly) interpolated and then executed as a system command with /bin/sh or its equivalent.`. So I guess it spawns `cmd.exe /c $command`, cmd.exe prints error message that `$command` was not recognized and exit...

Reply:
I fear that the error message can be localized in MSWin :fearful:

Reply:
Looks like that `system { $command } $command` does not work when `$command` is batch file without specified extension. Perl always throw exec warning `Can't spawn "cmd.exe": No such file or directory`. Return value of `system` function, `$?` and `${^CHILD_ERROR_NATIVE}` is 65280, `$!` and `$^E` is 2.

`system($command)` returns for non-existing command value 256, `$?` and `${^CHILD_ERROR_NATIVE}` is set to 256, `$!` and `$^E` is set to 2.

`system($command)` for existing command returns value `$exitcode << 8`, also `$?` and `${^CHILD_ERROR_NATIVE}` are set to this value. `$!` and `$^E` are set to 2 too!

So it is not possible to distinguish between command does not exist and command returned exit code 1.

Reply:
I changed this function to use `MM->maybe_command`. Seems it does right thing.